media/css/firefox/browsers/mobile.scss

```diff
@@ -167,6 +167,10 @@ $image-path: '/media/protocol/img';
         .c-page-header::before {
             left: 15%;
         }
+
+        :dir(rtl) .c-page-header::before {
+            right: 15%;
+        }
```

Comment:
So why not nesting it somehow, or `:dir(rtl) & {` shorthand etc. — Thing is I need the pseudo-selector somewhere between the first class (targeting the root `.android, .ios` a few blocks up) and this actual `::before` selector — as just chaining it up to it won't necessarily work as the pseudo-element itself has no content so is not actually RTL, to match the rule to apply.

I'd like to avoid such repetition, so if you have better tricks for this I'm all ears!